File: src/main/java/org/ventanas/Java/Ejercicio3.java
package org.ventanas.Java;
// 3. Cálculo del área de un círculo
// Pedir al usuario el radio de un círculo y calcular su área (A = π * r^2).
import java.util.Scanner;

public class Ejercicio3 {
    public static void execute() {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Dame el radio de tu círculo: ");
        try {
            double num = scanner.nextDouble();
            double area = Math.PI * (num * num);
            System.out.println("El área de tu círculo es de: " + area);
        } catch (Exception e) {
            System.out.println("Error al insertar el número, debe ser un valor numérico.");
        }
    }
}

File: src/main/java/org/ventanas/Kotlin/Ejercicio3.kt
package org.ventanas.Kotlin
// 3. Cálculo del área de un círculo
// Pedir al usuario el radio de un círculo y calcular su área (A = π * r^2).

fun ejercicio3() {
    print("Dame el radio de tu circulo: ")
    try {
        val num = readln().toDouble()
        val area = kotlin.math.PI *(num * num)
        println("El area de tu circulo es de: $area")



    }catch (e:Exception){
        println("Error al insertar el numero, debe de ser entero.")
    }


}